sites/firehouse.com/config/gam.js

```diff
@@ -2,6 +2,14 @@ const configureGAM = require('@endeavor-business-media/package-shared/config/gam
 
 const config = configureGAM({ basePath: 'Firehouse' });
 
+// Remove this block to convert to true via global gam config
+config.lazyLoad = {
```

Comment:
@Alixers505  you should be able to do this with just config.lazyLoad.enabled = false.

One concerned is that could affect delivered vs viewed ad impressions as no all lazy load image from load more will be requested on page load, no viewed.  Is the group/PM aware of this.

Reply:
I will forward the concern to Clint. He messaged the request to me yesterday.

Reply:
Do you want me to push up a correction PR with the single-line version of the change?